### ISSUE TITLE
Update user documentation

### ISSUE THREAD

**Issue by fritzophrenic** [fireout/keepasssequencer#10]
To be more user-friendly, we need a "getting started"/user guide in the readme, with documentation of each configuration item, and usage examples.

**Comment by fireout** [comment]
I changes this ticket to "update documentation" so that the screenshot represent the upcoming look.

**Comment by fritzophrenic** [comment]
Do you plan to still provide a default options.xml file in releases or can that sentence be removed from https://github.com/fireout/keepasssequencer/wiki/1.-Installation ?

**Comment by fireout** [comment]
We could probably have some sample configuration available, but with the default throwaway values, the options.xml is not mandatory, I would remove this line for now until we add sample templates

**Comment by fritzophrenic** [comment]
[File menu](https://github.com/fireout/keepasssequencer/wiki/2.1-Basic-Configuration%E2%80%94File-menu) and [Sequence editor](https://github.com/fireout/keepasssequencer/wiki/2.4-Basic-Configuration%E2%80%94Sequence-Editor) pages need an update with new screenshots with both New General Service List variants shown. Current images are from the old menu with only NGSL entry.

**Comment by fireout** [comment]
Definitly.

For future documentation changes requirements though, I think it would be easier to have the changes part of any ticket that triggers the change, rather than leaving this ticket open... ~~forever~~ until development stops :) .

**Comment by fritzophrenic** [comment]
Agree. I just noticed this ticket when I went to go write a new one. I was at work when I noticed so I didn't want to spend the time updating them immediately and didn't want to forget :-)
